go/v1beta1/storage/filtering/filtering_test.go

```diff
@@ -0,0 +1,84 @@
+package filtering
+
+import "testing"
+
+func TestSingleTerm(t *testing.T) {
+	filter := `a=="b"`
+	expectedResult := `{"query":{"bool":{"must":[{"term":{"a":"b"}}]}}}`
+	actualResult := ParseExpressionEntrypoint(filter)
+	if actualResult != expectedResult {
```

Comment:
instead of this `if` check every time, you can use the `assert` package, or you can still use the gomega assertions since we're already pulling them in as a dep anyways.